File: lista2/Ex1.java
package lista2;

import java.util.Scanner;

public class Ex1 {
    public static void main(String[] args) {

        Scanner input = new Scanner(System.in);

        int tempoMin;

        System.out.print("Digite o tempo em minutos: ");
        tempoMin = input.nextInt();

        System.out.println(tempoMin+" minuto(s) equivale(m) a "+tempoMin*60+" segundos");
    }
}

File: lista2/Ex1.c
#include <stdio.h>

int main() {

    int tempoMin;

    printf("Digite o tempo em minutos: ");
    scanf("%d", &tempoMin);

    printf("%d minuto(s) equivale(m) a %d segundos\n", tempoMin, tempoMin*60);

    return 0;
}